packages/react/src/hooks/useAuth.ts

```diff
@@ -31,6 +32,7 @@ type UseAuthReturn =
       orgSlug: null;
       signOut: SignOut;
       getToken: GetToken;
+      authStateKey: string;
```

Comment:
@raptisj Have we settled on the name in the design doc?

Reply:
@SokratisVidros We settled on this `authStateKey`. I saw your comments on the doc and it kinda makes sense too. Do you find `authStateKey` to be problematic?